src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusMessagePumpOptionsBase.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using GuardNet;
+
+namespace Arcus.Messaging.Pumps.ServiceBus.Configuration 
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class AzureServiceBusMessagePumpOptionsBase
```

Comment:
What is the benefit of not adding this to `AzureServiceBusMessagePumpOptions`?

I don't like Base suffix, it's discouraged by .NET design guidelines as well
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/base-classes-for-implementing-abstractions

Reply:
I think it's because the other one is used by pump internally. Then I would name this AzureServiceBusPumpOptions or whatever

Reply:
"ConfigurationOptions"...